src/latency.c

```diff
@@ -358,18 +356,17 @@ sds createLatencyReport(void) {
     }
 
     if (eventnum == 0 && advices == 0) {
-        report = sdscat(report, "Dave, no latency spike was observed during the lifetime of this Redis instance, not "
-                                "in the slightest bit. I honestly think you ought to sit down calmly, take a stress "
-                                "pill, and think things over.\n");
+        report = sdscat(report, "No latency spike was observed during the lifetime of this Valkey instance, not "
+                                "in the slightest bit.\n");
     } else if (eventnum > 0 && advices == 0) {
         report =
-            sdscat(report, "\nWhile there are latency events logged, I'm not able to suggest any easy fix. Please use "
-                           "the Redis community to get some help, providing this report in your help request.\n");
+            sdscat(report, "\nThere are latency events logged, they are not easy to fix. Please get "
```

Comment:
```suggestion
            sdscat(report, "\nThere are latency events logged that are not easy to fix. Please get "
```

Either like this or use two separate sentences: There are latency events logged. They are not easy to fix."

Reply:
Thanks for your suggestion. I will fix it.